### DISCUSSION TITLE
behind-the-scenes-of-minimal-apis-1-a-first-look-behind-the-scenes-of-minimal-api-endpoints/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#101] [Announcements]
# behind-the-scenes-of-minimal-apis-1-a-first-look-behind-the-scenes-of-minimal-api-endpoints/

In this post I take a high-level look at how a call to MapGet("/", () => "Hello World!") is turned into a RequestDelegate that ASP.NET Core can execute.

https://andrewlock.net/behind-the-scenes-of-minimal-apis-1-a-first-look-behind-the-scenes-of-minimal-api-endpoints/

**Comment by mauler2025** [comment]
Quoted from this article.
> Together they remove a lot of the ceremony associated with a generic-host based ASP.NET Core app using MVC.

Does it mean that Minimal Api does not use generic-host?

**Reply by mauler2025** [reply]
Oh. I see after reading whole. Question cancelled.
